plugins/CoreVisualizations/Visualizations/Cloud.php

```diff
@@ -194,7 +195,7 @@ private function getPercentage($popularity, $maxValue)
             return 0;
         }
 
-        $percent = ($popularity / $maxValue) * 100;
+        $percent = Piwik::getQuotientSafe($popularity, $maxValue, 4) * 100;
```

Comment:
This actually changes the resulting precision of the number.
Also it actually only covers the real problem why the calculation fails. I'll add a comment to the original issue around that.

Reply:
My thinking here is that the percent is being used to decide the size of the words in the cloud visualization, the original unsafe division wasn't applying a precision and four digit precision should be enough accuracy to differentiate between words.

It looks like this is too simplistic an approach and would be better fixed in #19940 - I'll close this PR as it was only intended as a possible quick fix.